pkgs/tools/security/otpgen/default.nix

```
@@ -0,0 +1,52 @@
+{ 
+  lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , fetchpatch
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
```

Typically, we format this as

```suggestion
{ lib
, stdenv
, makeWrapper
, fetchFromGitHub
, fetchpatch
, cmake
, pkg-config
, cryptopp
, qtbase
```

or less commonly

```suggestion
{
  lib,
  stdenv,
  makeWrapper,
  fetchFromGitHub,
  fetchpatch,
  cmake,
  pkg-config,
  cryptopp,
  qtbase,
```

---

New format here https://github.com/NixOS/nixpkgs/pull/87887/commits/c47c42adbf6b03724ae21117c50ae2c895a9518a